src/main/java/net/minecraftforge/coremod/api/ASMAPI.java

```diff
@@ -33,17 +32,29 @@ public static MethodNode getMethodNode() {
         return new MethodNode(Opcodes.ASM9);
     }
 
-    // Terribly named method. Should be called "prependMethodCall" or something similar.
     /**
      * Injects a method call to the beginning of the given method.
      *
      * @param node       The method to inject the call into
      * @param methodCall The method call to inject
      */
-    public static void appendMethodCall(MethodNode node, MethodInsnNode methodCall) {
+    public static void injectMethodCall(MethodNode node, MethodInsnNode methodCall) {
         node.instructions.insertBefore(node.instructions.getFirst(), methodCall);
     }
 
+    /**
+     * Injects a method call to the beginning of the given method.
+     *
+     * @param node       The method to inject the call into
+     * @param methodCall The method call to inject
+     *
+     * @deprecated Renamed to {@link #injectMethodCall(MethodNode, MethodInsnNode)}
+     */
+    @Deprecated(forRemoval = true, since = "5.1")
```

Comment:
I've marked this method as deprecated for removal, but should it be removed? It's existed in CoreMods for a long time and a lot of older CoreMods, some of which might still be used today in FMLOnly-type mods, might be using this method.